lib/generate_letters.ml

```diff
@@ -38,5 +38,104 @@ let rec index_of_letter curr letter = function
   | h :: t ->
       if h = letter then curr else index_of_letter (curr + 1) letter t
 
-(** used http://www.breakingthegame.net/leaves2 to generate accurate
-    probability rates for vowels and consonants*)
+(* vowels is a list containing every vowel char accepted by our Scrabble
+   game*)
+let vowels = [ 'A'; 'E'; 'I'; 'O'; 'U' ]
+
+(* consonants is a list containing every consonant char accepted by our
+   Scrabble game *)
+let consonants =
+  [
+    'B';
+    'C';
+    'D';
+    'F';
+    'G';
+    'H';
+    'J';
+    'K';
+    'L';
+    'M';
+    'N';
+    'P';
+    'Q';
+    'R';
+    'S';
+    'T';
+    'V';
+    'W';
+    'X';
+    'Y';
+    'Z';
+  ]
+
+(* probability weights associated with each number of vowels in a
+   starting hand. For implementer reference.*)
+(**let weights =
+   [ (0, 0.07); (1, 0.25); (2, 0.36); (3, 0.24); (4, 0.07); (5, 0.01) ]*)
+
+(* alias table reference https://www.keithschwarz.com/darts-dice-coins/*)
+(* probability list needed for an alias table*)
+let prob = [ 0.42; 0.92; 0.56; 1.; 0.42; 0.06 ]
+
+(* alias list needed for an alias table*)
+let alias = [ 1; 2; 3; 0; 2; 2 ]
+
+(** [biased prob total] is true for heads and false for tails. This
+    method simulates a biased coin, with a [prob]/[total] chance of
+    being heads. Requires [prob] <= [total].*)
+let biased prob total =
```

Comment:
`biased` is not a name that makes it easy to understand what this function does without reading the comment. consider renaming to `random_binary`, `random_bool`, or something like that?